automation/check-merged.environment.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Can we remove this and use travis like all our projects?

This PR https://github.com/kubevirt/bridge-marker/pull/5 should do it already.

Reply:
Removed